src/task_scl_eff_pot_hab.py

```diff
@@ -162,6 +116,29 @@ def density_to_patch_size(ft):
                 None, {"ECO_ID": ft.get("ECO_ID"), "min_core_size": min_core_size}
             )
 
+        def dilate(image, distance):
+            dialated_image = image.fastDistanceTransform(distance).sqrt()
+            return dialated_image.lte(distance).selfMask()
+
+        elev_mask = self.elevation.lt(self.thresholds["elev"]).selfMask()
+        str_hab_mask = self.structural_habitat.gt(self.thresholds["str_lc"]).selfMask()
+        current_range = self.historic_range.updateMask(self.extirpated).selfMask()
+        low_hii_mask = self.hii.lte(self.thresholds["hii"]).selfMask()
+        str_hab = (
+            str_hab_mask.updateMask(elev_mask)
+            .updateMask(low_hii_mask)
+            .updateMask(self.water)
+            .updateMask(current_range)
+        )
+
+        connected_potential_habitat = str_hab.connectedPixelCount(625, True).reproject(
```

Comment:
Should `625` be a var so we know why that many pixels?

Reply:
yes, and in fact it already was programmed as a variable